src/args.js

```diff
@@ -1,9 +1,5 @@
 import yargs from 'yargs'
 import { partialRight, omitBy, mapKeys, compact } from 'lodash'
-import { camelCase } from 'change-case'
-import QS from 'qs'
```

Comment:
Looks to me like you can remove `change-case` and `qs` from the project dependencies now :+1:

Reply:
👍

Reply:
done, thank you